### PR TITLE
PCM-3145: restoring chat functionality

### PR DESCRIPTION
**Pull request by engineersamuel**
Fixing chat functionality per https://projects.engineering.redhat.com/browse/PCM-3145

### CODE REVIEW

**Comment by engineersamuel** [comment]
@jtrantin @gandhikeyur @vrathee Please review.  I have verified locally that I can connect to chat and exchange messages with the qa livegent.  I have not verified other environments that I don't have access to but locally it works.  QA will need to verify the other environments.

File: app/chromed.ejs [hunk 1]
```diff
@@ -34,9 +34,11 @@
     </div>
 </div>
 
-<!-- TODO What is peg and where is it used? -->
-<!-- window.deps.push("/bower_components/pegjs/peg-0.9.0.js");-->
-<script type="text/javascript" src="https://c.la8cs.salesforceliveagent.com/content/g/js/31.0/deployment.js"></script>
+<% if (htmlWebpackPlugin.options.isProd) { %>
+    <script type="text/javascript" src="https://c.la1w1.salesforceliveagent.com/content/g/js/31.0/deployment.js"></script>
+<% } else { %>
```
Comment (engineersamuel) [inline app/chromed.ejs]: Is this right for production?

File: app/main.module.js [hunk 2]
```diff
@@ -160,10 +160,46 @@ if (ENVIRONMENT === 'gs4') {
...
+                        // CHAT_SUPPORT.chatInitHashTwo = '00DK000000W3mAm';
+                        // The dev-rogsstest url returns a 404
+                        // CHAT_SUPPORT.chatIframeHackUrlPrefix = 'https://dev-rogsstest.cs9.force.com/chatHidden';
+
+                        // Taken from https://github.com/redhataccess/redhat_access_angular_ui/blob/1.1.93/app/redhat_access_module.js
+                        CHAT_SUPPORT.chatButtonToken = '573A0000000GmiP';
+                        CHAT_SUPPORT.chatLiveAgentUrlPrefix = 'https://d.la6cs.salesforceliveagent.com/chat';
+                        CHAT_SUPPORT.chatInitHashOne = '572A0000000GmiP';
+                        CHAT_SUPPORT.chatInitHashTwo = '00DJ0000003OR6V';
+                        CHAT_SUPPORT.chatIframeHackUrlPrefix = 'https://qa-rogsstest.cs10.force.com/chatHidden';
```
Comment (engineersamuel) [inline app/main.module.js]: This is the section that gets hit in local development.

**Comment by jtrantin** [comment]
I have very little knowledge of the chat functionality, but I don't see any problems.

**Comment by renujhamtani** [comment]
this is looking good. QA needs to test on all environments. I will put this comment in JIRA.

**Pull request merged**